### PR TITLE
Responsiveness to the Website

### LINKED ISSUE #1: Search algos not responsive
The searching algorithm is not responsive and does not fit the screen on mobile screen.

### PR DESCRIPTION
**Pull request by jainaayush01**
Changes made are:
Added Timeline.
Fixed Navbar issue in search page.
Removed extra div elements.
Improved codebase readability
Responsiveness for both the pages
Improved Structure and style of the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,41 +21,22 @@
         </div>
```
Comment (akshitadixit) [inline index.html:21]: Could you activate gh-pages on your repo and allow me to see the live version of the changes that are made here?

File: sortingAlgos.js [hunk 2]
```diff
@@ -1,21 +1,23 @@
 function bubbleSort() {
```
Comment (akshitadixit) [inline sortingAlgos.js]: Thank you for the minute changes here, this was default formatting added by vscode.

File: temp/style.css [hunk 3]
```diff
@@ -1,51 +1,51 @@
 section {
```
Comment (akshitadixit) [inline temp/style.css:1]: the files in the temp folder were temporary and simply used to store unused/test code

**Comment by akshitadixit** [comment]
Things look fine and good to go. Please check for the reviews. Also a recent merge has been made, kindly update your fork with the same.

File: index.html (hunk 1 above)
Comment (jainaayush01) [reply index.html:21]: Okay

File: sortingAlgos.js (hunk 2 above)
Comment (jainaayush01) [reply sortingAlgos.js]: Welcome

File: temp/style.css (hunk 3 above)
Comment (jainaayush01) [reply temp/style.css:1]: Yeah i saw that earlier while I was changing the temp/style.css and nothing was changing so yeah.

**Comment by jainaayush01** [comment]
Okay.

**Comment by jainaayush01** [comment]
I have updated the fork, applied the small naming convention and GitHub-pages is also active.
Have a look here: https://jainaayush01.github.io/Structurex/

**Review by akshitadixit — APPROVED** [review]
Perfect, thanks.

**Pull request merged**